### DISCUSSION TITLE
为啥我的docker部署的后端显示 Connection refused

### DISCUSSION THREAD

**Discussion by ZikiP** [metatube-community/jellyfin-plugin-metatube#160] [Q&A]
早上用的好好的，下午就炸了。显示拒绝连接，用的节点是日本的，求助

**Comment by MonkeySan186** [comment]
mogenius部署后端也显示“The service crashed during the build process”，是不是不能用了

**Reply by xjasonlyu** [reply]
不清楚，可以考虑自建的，这些平台似乎已经不够稳定了。

**Reply by ZikiP** [reply]
我本来是自建的。但是好像对节点要求很高，搜元数据很慢
